src/libNode/DSBlockProcessing.cpp

```diff
@@ -528,23 +530,43 @@ bool Node::ProcessDSBlock(const vector<unsigned char>& message,
             m_mediator.m_ds->ProcessTxnBodySharingAssignment();
 
             // Update my DS mode and ID
-            m_mediator.m_ds->m_consensusMyID = 0;
             m_mediator.m_ds->m_consensusID
                 = m_mediator.m_currentEpochNum == 1 ? 1 : 0;
-            m_mediator.m_ds->m_mode = DirectoryService::Mode::PRIMARY_DS;
 
-            // (We're getting rid of this eventually) Clean up my txn list since I'm a DS node now
             m_mediator.m_node->CleanCreatedTransaction();
 
-            LOG_EPOCHINFO(to_string(m_mediator.m_currentEpochNum).c_str(),
-                          DS_LEADER_MSG);
-            LOG_STATE("[IDENT]["
```

Comment:
Should we keep this log message?

Reply:
The point is this node might not be the DS Leader, there is a further condition. check line 552.

Reply:
I meant should we also move the `LOG_STATE` message into the appropriate location.  Seems like only the `LOG_EPOCHINFO` one was retained.

Reply:
okay, noted.